recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "5.5" %}
+{% set version = "6.1.1" %}
 
 package:
   name: coverage
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/coverage/coverage-{{ version }}.tar.gz
-  sha256: ebe78fe9a0e874362175b02371bdfbee64d8edc42a044253ddf4ee7d3c15212c
+  sha256: b8e4f15b672c9156c1154249a9c5746e86ac9ae9edc3799ee3afebc323d9d9e0
 
 build:
   number: 2
```

Comment:
Reset build number to `0`

Reply:
Add skip: True. # [py<36]
see https://github.com/nedbat/coveragepy/blob/7a0188231be8f7bb04b2832365401062e18b7ab2/setup.py#L135

Reply:
reset the build number and added the skip section:

https://github.com/AnacondaRecipes/coverage-feedstock/blob/update_to_6.1.1/recipe/meta.yaml#L12&L15